### DISCUSSION TITLE
Transfer to physical robot

### DISCUSSION THREAD

**Discussion by HeChengHui** [aidudezzz/deepbots#99] [Q&A] [answered]
Hi, I managed to get the emitterReceiverSchemeTutorial working and was wondering how to transfer the completed model into a physical robot since it was not covered in the tutorial.

**Comment by tsampazk** [comment, marked as answer]
Hey @HeChengHui , sorry for the long time it took to answer your question.

Transfering to a physical robot is currently out of scope of the deepbots library. It is highly dependant on the use-case and the physical setup you have. 

What we provide is the assurance that by using the emitter/receiver scheme, the robot is as-realistic-as-possible, meaning it doesn't have access to information and data it shouldn't be able to have if it were a real robot, e.g. x,y coordinates without using a GPS sensor or similar, Euclidean distance to a target, etc.

Moreover, the emitter/receiver setup can be analogous to a setup where for example you use a laptop to run the supervisor code, neural network model, etc. and communicate the actions via bluetooth to a physical robot. Of course, the bluetooth communication code will be different than what we use in Webots, but we hope that depending on your choice of setup and libraries you can easily translate the code from Webots-specific to your-setup-specific.
